certified-connectors/InEight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3050 @@
+              },
+              "x-powered-by": {
+                "description": "x-powered-by",
+                "type": "string"
+              },
+              "x-requestid": {
+                "description": "x-requestid",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "UserRoles_Import",
+        "description": "UserRoles Integration allows InEight customers to import a list of users and roles mapping at the intended project or organization level. Within the application, user role assignments are found in the Account level menu options under \"Suite Administration > User management > Edit user\".",
+        "operationId": "UserRoles_Import",
+        "parameters": [
+          {
+            "name": "X-IN8-TENANT-PREFIX",
+            "in": "header",
+            "description": "The tenant-prefix of your Project Suite base URL. For example, if you are accessing Project Suite at *https://sample-domain.hds.ineight.com/* then the value for this parameter would be **sample-domain**",
+            "x-ms-summary": "The tenant-prefix of https://t01.hds.ineight.com/ would be \"t01\"",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "Ocp-Apim-Subscription-Key",
+            "in": "header",
+            "description": "This is found in in your APIM Profile.",
+            "x-ms-summary": "This is found in in your APIM Profile.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "required": [
+                  "RoleName",
+                  "UserSourceSystemId"
+                ],
+                "properties": {
+                  "RoleName": {
+                    "type": "string",
+                    "description": "RoleName"
+                  },
+                  "UserSourceSystemId": {
+                    "type": "string",
+                    "description": "UserSourceSystemId"
+                  },
+                  "IsActive": {
+                    "type": "boolean",
+                    "description": "IsActive"
+                  },
+                  "ProjectSourceSystemId": {
+                    "type": "string",
+                    "description": "ProjectSourceSystemId"
+                  },
+                  "OrganizationSourceSystemId": {
+                    "type": "string",
+                    "description": "OrganizationSourceSystemId"
+                  }
+                }
+              }
+            }
+          }
+        ],
+        "consumes": [
+          "application/json",
+          "text/plain"
+        ],
+        "produces": [
+          "application/json",
+          "text/plain"
+        ]
+      }
+    },
+    "/''": {}
```

Comment:
@InEightDevOps Can you please let us know the reason to add this line of code and its output? If it is a typo, please remove it.